drivers/Makefile.dep

```diff
@@ -72,6 +72,11 @@ ifneq (,$(filter ltc4150_%,$(USEMODULE)))
   USEMODULE += ltc4150
 endif
 
+ifneq (,$(filter mcp2515,$(USEMODULE)))
+  USEMODULE += xtimer
+  FEATURES_REQUIRED = periph_gpio periph_spi periph_gpio_irq
```

Comment:
```suggestion
  FEATURES_REQUIRED += periph_gpio periph_spi periph_gpio_irq
```

I think this is the issue with a bunch of murdock issues :) (probably most)

Reply:
Oh wow... I would never have found that! 
Let's see if Murdock is happy this time :)